### DISCUSSION TITLE
AlDente 1.18

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#755] [General]
* Added official support of AlDente Pro to 16 more Intel based MacBooks. Please take a look at the following blog post for a full list of all newly supported MacBook models: https://apphousekitchen.com/adding-aldente-pro-support-to-16-new-intel-macbook-models-full-list-below/
* New feature! Start of the beta phase for controlling AlDente Pro via the macOS Shortcuts app. Read more about this new feature here: https://apphousekitchen.com/feature-explanation-shortcuts/
* AlDente now locks the screen automatically if you close your MacBook while sleeping is disabled
* Fixed a bug where the “Disable optimised Charging” message would show up on every launch of AlDente
* Fixed a bug where sleep would completely disable without being necessary under certain conditions while discharging
* Fixed a bug where Calibration Mode would not work correctly when “Hardware Battery Percentage” was enabled

​
With peace & love from Vienna,
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente/releases/tag/1.18'>AlDente 1.18</a>.</em>

**Comment by TheImpendingDoom** [comment]
"AlDente now locks the screen automatically if you close your MacBook while sleeping is disabled" How do you disable this?

I use a second MBP as a remote machine via Jump Desktop and have it setup via terminal commands to stay awake even when the lid is closed (never go to sleep). It is in clamshell mode in a shelf in my desk. Now this feature causing the login screen to reappear every 2 seconds. Basically I cannot use it the way I want to.
